src/app/store/slices/storage/storage.thunks/uploadFolderThunk.ts

```diff
@@ -296,6 +307,21 @@ function countItemsUnderRoot(root: IRoot): number {
 
   return count;
 }
+function getItemsSize(root: IRoot): number {
+  let size = 0;
+
+  const queueOfFolders: Array<IRoot> = [root];
+
+  while (queueOfFolders.length) {
```

Comment:
`while (queueOfFolders.length > 0)` is better to understand, I am not sure about the side effects of checking is the `length` attribute is present